src/twisted/conch/test/test_filetransfer.py

```diff
@@ -535,18 +535,23 @@ def test_openDirectoryIterator(self):
 
```

Comment:
The comment above states "before waiting on them via yield", but that `yield` statement is now `await`.